dev/Microsoft.Process.Environment/EnvironmentManager.h

```diff
@@ -29,6 +29,13 @@ namespace winrt::Microsoft::ProjectReunion::implementation
 
     private:
         Scope m_Scope;
+
+        std::wstring USER_EV_REG_LOCATION = L"Environment";
```

Comment:
Does this need to be a wstring? If this is just a constant just make it PCWSTR

or maybe `inline constexpr std::wstring_view blah{ L"Environment" };. There was a recent discussion of how to most efficiently declare null-terminated string constants (wstring_view isn't guaranteed to be null-terminated e.g. `std::wstring_view(std::wstring(L"Foo"), 0, 1);` but in this case it is). There were a dozen+ notations discussed and 2-3 were correct and most efficient, but I don't recall which was best.

@jonwis @oldnewthing do you recall? What's the recommendation for now (until the coding guidelines are updated)?